### ISSUE TITLE
Please update the documentation to reflect new API call: /revokeSignInSessions

### ISSUE THREAD

**Issue by tfrancois** [microsoftgraph/microsoft-graph-docs#6682]
The online documentation has not been updated to reflect the breaking change that was reported by Microsoft and went into effect in March 2019.  The correct call should be /revokeSignInSessions, which replaces the deprecated (and now invalid)  /invalidateAllRefreshTokens.

More info in confirmation thereof:
https://developer.microsoft.com/en-us/office/blogs/breaking-change-invalidate-all-refresh-tokens-update-in-microsoft-graph-beta/

** EDIT **:
Having made the correction to my code, I'm still getting an error when I follow the instructions as provided above.  The error response I get is:

```
{ error:
   { code: 'BadRequest',
     message:
      'Write request id : action:1 does not contain Content-Type header or body.',
     innerError:
      { 'request-id': 'd5522155-bebd-43e1-a79b-4801e31f9437',
        date: '2019-12-26T06:31:53' } } }
```
Can someone please look into?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0b3f5dd2-e23e-4453-258d-036950e5a589
* Version Independent ID: 2290444e-dbb1-194f-d6f5-132972153998
* Content: [user: invalidateAllRefreshTokens - Microsoft Graph beta](https://docs.microsoft.com/en-us/graph/api/user-invalidateallrefreshtokens?view=graph-rest-beta&tabs=http)
* Content Source: [api-reference/beta/api/user-invalidateallrefreshtokens.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/master/api-reference/beta/api/user-invalidateallrefreshtokens.md)
* Product: **microsoft-identity-platform**
* Technology: **microsoft-graph**
* GitHub Login: @dkershaw10
* Microsoft Alias: **MSGraphDocsVteam**

**Comment by tfrancois** [comment]
The issue eventually worked when I added the Content-Type: 'application/json'  and an empty JSON body.  This however is not listed as a requirement anywhere in the existing docs or the post above discussing the breaking change.  Please advise.

**Comment by dkershaw10** [comment]
Thanks - will update the docs.

**Comment by chuyingqifei** [comment]
Hi, I found the same issue here while trying to revoke signed in session using "POST https://graph.microsoft.com/beta/me/invalidateAllRefreshTokens". 

And I changed to use revokeSignInSessions api by referring to https://developer.microsoft.com/en-us/office/blogs/breaking-change-invalidate-all-refresh-tokens-update-in-microsoft-graph-beta/ and found that the returned StatusCode is 200 rather than 204, should we update the doc for the returned StatusCode as well?

Also after the successful API call, I found that the aad user can still access the organization resources without having to refresh the browser if the user keeps operating.  I wonder if we could update the doc for expected outcome with the API?

Thanks,
Xiaobo Kang(xika@microsoft.com)
